packages/web3-core-method/src/index.js

```diff
@@ -561,7 +561,7 @@ Method.prototype.buildCall = function() {
         };
 
         // Send the actual transaction
-        if(isSendTx && _.isObject(payload.params[0]) && !payload.params[0].gasPrice) {
+        if(isSendTx && _.isObject(payload.params[0]) && payload.params[0].gasPrice === undefined) {
```

Comment:
better is `typeof payload.params[0].gasPrice === 'undefined'`

Reply:
I agree!

Reply:
@frozeman I changed comparison logic through 'typeof' operator.